internal/store/peering_connections.go

```diff
@@ -30,6 +30,7 @@ type PeeringConnectionLister interface {
 
 type PeeringConnectionCreator interface {
 	AzureContainers(string) ([]atlas.Container, error)
+	AWSContainers(string) ([]atlas.Container, error)
 	CreateContainer(string, *atlas.Container) (*atlas.Container, error)
 	CreatePeeringConnection(string, *atlas.Peer) (*atlas.Peer, error)
 }
```

Comment:
I'm already doing this for GCP but let's split this
`type AzurePeeringConnectionCreator interface {` and `type AWSPeeringConnectionCreator interface {` and this way they don't share the List method